libs/docs/shared/NEW_COMPONENT.md

```diff
@@ -1,99 +1,249 @@
+    font-size: 16px;
+    margin-bottom: 20px;
+}
+```
+If you start the documentation app now, you should see the following result in `My Component`:
+
+<img width="1640" alt="Screenshot 2023-08-09 at 9 11 02 AM (2)" src="https://github.com/SAP/fundamental-ngx/assets/132930816/5596c431-cff6-4237-b46d-2dc45ae73f34">
+
+## <a name="5"></a>5. Updating Documentation
+
+In the following section of the report, we will see how to update the documentation app, regarding our newly created component.All the required files can be found at `libs/docs/core`.
+
+The html for the component page is split into two parts. The header which contains a brief decrepitation of component, and the example section which contains some relevant use cases for the component.
+The header section can be found in the file `[component name]-header`.
+
+
+While the folder contains a component declaration, you will mainly be working with only the html file. In it it should have the following code :
+
+```html
+<fd-doc-page>
+    <header>My Component</header>
+
+    <description>
+        <!-- TODO: description -->
+    </description>
+
+    <import module="MyComponentModule" subPackage="my-component"></import>
+
+    <fd-header-tabs></fd-header-tabs>
+</fd-doc-page>
 ```
-libs/docs/core/poster-editor/examples/
-        /default
-            ....
-        /complex
-            poster-editor-complex-example.component.html
-            poster-editor-complex-example.component.ts
+Any general information should be writen between the `description` declaration.
+
+```html
+<fd-doc-page>
+    <header>My Component</header>
+
+    <description>
+        This is my first component. When you press it will inform you.
+    </description>
+
+    <import module="MyComponentModule" subPackage="my-component"></import>
+
+    <fd-header-tabs></fd-header-tabs>
+</fd-doc-page>
+```
+
+We will now look at the example folder ,`examples`, which is used to contain all the created examples for the functionality of the component. You will notice that the folder is not empty, as a default example was created when generating the component. To demonstrate how to add more examples, we will create a new one, that will contain two copies of the button created previously. Firstly, we will create a folder that has a relevant name to the example. The most utilized naming conventions here are  `[component name]-[example name]-example`, and `[example-name]-example,where all the words are separated by a dashed line. As for the "example name", the most discriptive, and short one, should be used, for example since our example will contain two instances of our button component, we can just call it `double-button`, so that it clearly communicates to other users what it is demonstrating. For the naming conventions, while both are equally popular, a common rule of thumb is to use the former for components with shorter names, while the latter for components with longer or more complex names.
```

Comment:
Some formatting oddities in here like spaces before commas, which I think should be fixed by linting. Also I think "where all the words are separated by a dashed line. As for the "example name", the most discriptive, and short one, should be used, for example since our example will contain two instances of our button component, we can just call it" is formatted as if it is code, missing a back quotation mark somewhere